task/common/machine/script.go

```diff
@@ -32,6 +32,13 @@ base64 --decode << END | sudo tee /usr/bin/tpi-task > /dev/null
 END
 chmod u=rwx,g=rx,a=rx /usr/bin/tpi-task
 
+sudo tee /usr/bin/tpi-task-shutdown << 'END'
+#!/bin/bash
+
+(systemctl is-system-running | grep stopping) || tpi --stop;
+END
+chmod u=rwx,g=rwx,o=rwx /usr/bin/tpi-task-shutdown
```

Comment:
```suggestion
chmod u=rwx,g=rx,o=rx /usr/bin/tpi-task-shutdown
```

💅🏼

Reply:
Im wondering why the smoke tests are failing